ResumptionScenarios.md

```diff
@@ -0,0 +1,108 @@
+23                                                                                               | <- REQUEST_RESPONSE(Resume(0) StreamID(2)) // RPosition(0) Position(1) IPosition(0)
+24                                                                                               |NECTION~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+25                                                                                               ⍭ <~ PAYLOAD         (Resume(0) StreamID(1)) // RPosition(1) Position(4) IPosition(1)
+26 ~> PAYLOAD         (Resume(0) StreamID(3)) // RPosition(1) Position(2) IPosition(2)           ⍭
+27 ~> PAYLOAD         (Resume(0) StreamID(3)) // RPosition(1) Position(3) IPosition(2)           ⍭
+28------------------------------------------------------------------------------------------RECONNECTION----------------------------------------------------------------------------------
+29 -> RESUME          (NoE(2), { StreamID(1) : 1, StreamID(3) : 2 })                             |
+30                                                                                               | -> RESUME          (NoE(2), { StreamID(1) : 1, StreamID(3) : 2 })          
+31                                                                                               | <- RESUME_OK       (NoE(2), { StreamID(1) : 1, StreamID(3) : 1 })
+32                                                                                               | <- REDELIVER 3 FRAMES          
+33 <- RESUME_OK       (NoE(2), { StreamID(1) : 1, StreamID(3) : 2 })                             |
+33 <- RESUME_OK       (NoE(2), { StreamID(1) : 1, StreamID(3) : 2 })                             |
+34 -> REDELIVER 3 FRAMES                                                                         |
+35 <- PAYLOAD         (Resume(0) StreamID(1)) // RPosition(1) Position(1) IPosition(2)           |
+36 <- PAYLOAD         (Resume(0) StreamID(1)) // RPosition(1) Position(1) IPosition(3)           |
+37                                                                                               | -> PAYLOAD         (Resume(0) StreamID(3)) // RPosition(2) Position(2) IPosition(2)           
+38                                                                                               | -> REQUEST_STREAM  (Resume(0) StreamID(5)) // RPosition(0) Position(0) IPosition(1)           
+39                                                                                               | -> PAYLOAD         (Resume(0) StreamID(5)) // RPosition(2) Position(2) IPosition(3)           
+```
+
+according to the above sequence we can describe it as the following:
+<pre>
+<code>
+[<span style="color:red"><b>Line  3</b></span>][<span style="color:green"><i>Client</i></span>][StreamID(0)] sends Setup with Resumability enabled
```

Comment:
I think it might be good to have comments above their respective lines:
```
Client                                                                                         | Server 
1                                                                                                |
2                                                                                                |
      // send Setup with Resumability enabled
3  -> SETUP(ResumeFlag(1))                                                  |
4                                                                                               | -> SETUP
...
```